docker/docker_image.go

```diff
@@ -91,6 +91,12 @@ func GetRepositoryTags(ctx context.Context, sys *types.SystemContext, ref types.
 		}
 		for _, tag := range tagsHolder.Tags {
 			if _, err := reference.WithTag(dr.ref, tag); err != nil { // Ensure the tag does not contain unexpected values
+				// Per https://github.com/containers/skopeo/issues/2409 , Sonatype Nexus 3.58, contrary
+				// to the spec, may include JSON null values in the list; and Go silently parses them as "".
+				if tag == "" {
+					logrus.Debugf("Ignoring invalid empty tag")
```

Comment:
Would it make sense to show which tag was it? Might help with some debugging efforts.

Reply:
Albeit, not sure if at all possible to do so.

Reply:
We just get a `null` in an array; the server does not tell us why it included a `null`.

(We could, with some extra effort, get the raw `null` value instead of `""`, and be able to report whether we got one or the other, but I don’t think that is all that much more helpful. If we did want to include more details, I’d be tempted to just dump an escaped version of the response body as is, instead of trying to parse it.

Ultimately, this is all a debug-level log that users are not expected to see unless they are specially looking for it.)